### DISCUSSION TITLE
Install UPS conversion mod to items that still have a magazine inside it

### DISCUSSION THREAD

**Discussion by AniDemi** [cataclysmbn/Cataclysm-BN#2761] [General]
#### Issue
Currently to install the UPS conversion mod, one has to remove the magazine from the item and then activate the mod to install. However, when the player has a compatible tool with a magazine, trying to activate the mod will give this
![image](https://user-images.githubusercontent.com/60388907/236471181-c7bdca75-14e2-4c8b-ac28-64c4e33cf359.png)

#### Suggestion
Make the UPS conversion mod unload the item before installing the mod, so that the magazine is preserved and so that one can make it so that all compatible tools, even those with a magazine show up in this list
![image](https://user-images.githubusercontent.com/60388907/236472249-b032bb88-b148-41f2-a72d-64f5345e0a08.png)

**Comment by zantanzuken** [comment]
or, also an idea, have the UPS conversion mod incorporate the current battery into the ups system, like how UPS-type items that don't need the mod work. so it both has an internal charge and is rechargeable via UPS (but make the conversion fail if the battery inside is disposable)
